README.md

```diff
@@ -15,3 +15,38 @@ Run `cmake` to configure the build environment and then `make` to build
 To install, use `make install`
 
     sudo make install
+
+## Writing contract files
+Elementary OS's Pantheon's file manager supports adding options to the context menu by the way of Contract files.
```

Comment:
"elementary" is always lowercase. A much simpler way to refer to Files here is just "elementary Files" if your intent is to disambiguate. But Contractor menu actions also appear in Photos and other apps

Reply:
Good points.